### DISCUSSION TITLE
invalid domain name

### DISCUSSION THREAD

**Discussion by Lordkyllua** [domcloud/domcloud#27] [General]
Hello. Every time I try to create a host I get the error. invalid domain name. for what is this?

**Comment by willnode** [comment]
It might be because of confusing UX...

So when you create a new host, you need to tell the username and domain name.

The username is automatically generated by default (orange in the image below), and the domain name follows with `*.domcloud.io`. If you choose to use a custom domain name, you must tell the valid domain name (blue in the image below), (must have at least an LTD like *.com or something).

Note that a custom domain doesn't instantly make your website available. You have to own the domain and set some A/AAAA addresses to DOM Cloud Servers.

![image](https://user-images.githubusercontent.com/20214420/212850922-bb08df92-6db7-4ac1-8acb-a5dec46e7dc2.png)

**Reply by Lordkyllua** [reply]
![Screenshot_20230117-062324](https://user-images.githubusercontent.com/117127321/212859590-0704e5a6-bc3e-4479-a3d9-612193a4ba26.png)
I want to create a host with your domain.but this doesn't work for me that error always comes out

**Reply by Lordkyllua** [reply]
![Screenshot_20230117-062743](https://user-images.githubusercontent.com/117127321/212860613-51d0dad7-edfd-4742-81f9-d5cf9aae9c5d.png)

**Reply by willnode** [reply]
Ouch! ok my bad. Something wrong with the validator on the server side. Has been fixed now. Thanks for noticing!
